### DISCUSSION TITLE
error using the script to make a debian vm

### DISCUSSION THREAD

**Discussion by digitalgp** [tteck/Proxmox#2173] [General]
![image](https://github.com/tteck/Proxmox/assets/23147603/a37c7e78-79ad-421a-9469-a23695622fb9)

Hi i got this error using the script to make a debian vm when i tried to update the distro with apt-get upgrade, same with apt-get install docker to, no space left.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/1988
